migrate_config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import json
+import sys
+
+with open(sys.argv[1]) as fixture_file:
+    fixture = json.load(fixture_file)
+
+events = {}
+
+for obj in fixture:
+    if obj['model'] == 'django_emarsys.event':
+        events[obj['pk']] = obj
+        obj['params'] = {}
+
+for obj in fixture:
+    if obj['model'] == 'django_emarsys.eventdata':
+        event_obj = events[obj['fields']['event']]
+        event_obj['params'][obj['fields']['kwarg_name']] = obj['fields']
+
+event_config = {}
+
+for unused_event_id, event_obj in events.items():
+    print event_obj
+    event_config[event_obj['fields']['name']] = {
+        arg: (params['name'], '.'.join(params['content_type']))
+        for arg, params in event_obj['params'].items()
+    }
+
+print "EMARSYS_EVENTS = {"
+for event_name, params in sorted(event_config.items(),
+                                 lambda a, b: cmp(a[0], b[0])):
+    print u'    "{}": {{'.format(event_name.replace('"', '\"'))
```

Comment:
Yeah. I was going to suggest putting the code into a gist and linking to the gist in the commit message and/or the README.